sidebarsDocs.js

```diff
@@ -75,6 +75,18 @@ const sidebarsDocs = {
         'openstack-image-manager/contribute'
       ]
     },
+    {
+      type: 'category',
+      label: 'SCS Status Page',
```

Comment:
How important is it to prefix the status page in the sidebar with `SCS`as we are already in the SCS Scope?

Reply:
Not very important. In the docs, I just tended to refer to the status page as "SCS Status Page", so that the phrase can be used as identiying "name", not just for describing the type of software.

Should I change it?

Reply:
Done in 1775075.